bedwars-plugin/src/main/java/com/andrei1058/bedwars/levels/internal/LevelListeners.java

```diff
@@ -124,4 +126,35 @@ public void onArenaLeave(PlayerLeaveArenaEvent e) {
             if (pl != null) pl.updateDatabase();
         });
     }
+
+    @EventHandler
+    public void onBreakBed(PlayerBedBreakEvent e) {
+        Player player = e.getPlayer ();
+        if (player == null) return;
+        int beddestroy = LevelsConfig.levels.getInt("xp-rewards.bed-destroyed");
+        if (beddestroy > 0) {
+            PlayerLevel.getLevelByPlayer(player.getUniqueId()).addXp(beddestroy, PlayerXpGainEvent.XpSource.BED_DESTROYED);
+            player.sendMessage(Language.getMsg(player, Messages.XP_REWARD_BED_DESTROYED).replace("{xp}", String.valueOf(beddestroy)));
+        }
+    }
+
+    @EventHandler
+    public void onKill(PlayerKillEvent e) {
+        Player player = e.getKiller ();
+        Player victim = e.getVictim ();
+        if (player == null || victim.equals(player)) return;
```

Comment:
for statements use the java convention regarding brackets. we or I at least I do not like c style